lib/hexdocs/application.ex

```diff
@@ -19,6 +22,22 @@ defmodule Hexdocs.Application do
     Supervisor.start_link(children, opts)
   end
 
+  if Mix.env() == :prod do
+    defp goth_spec() do
+      credentials =
+        "HEXDOCS_GCP_CREDENTIALS"
+        |> System.fetch_env!()
+        |> Jason.decode!()
+
+      options = [scopes: ["https://www.googleapis.com/auth/devstorage.read_write"]]
+      {Goth, name: Hexdocs.Goth, source: {:service_account, credentials, options}}
```

Comment:
dunno if I asked in the past, if you have any feedback about this "new" way of using Goth, it'd be very appreciated.